docs/api/crash-reporter.md

```diff
@@ -12,49 +15,59 @@ crashReporter.start({
 });
 ```
 
-## crashReporter.start(options)
+## Methods
+
+The `crash-reporter` module has the following methods:
+
+### `crashReporter.start(options)`
 
-* `options` Object
-  * `productName` String, default: Electron
-  * `companyName` String, default: GitHub, Inc
-  * `submitUrl` String, default: http://54.249.141.255:1127/post
-    * URL that crash reports will be sent to as POST
-  * `autoSubmit` Boolean, default: true
-    * Send the crash report without user interaction
-  * `ignoreSystemCrashHandler` Boolean, default: false
+* `options` Object, properties:
+  * `productName` String, default: Electron.
+  * `companyName` String, default: GitHub, Inc.
+  * `submitUrl` String, default: http://54.249.141.255:1127/post.
+    * URL that crash reports will be sent to as POST.
+  * `autoSubmit` Boolean, default: `true`.
+    * Send the crash report without user interaction.
+  * `ignoreSystemCrashHandler` Boolean, default: `false`.
   * `extra` Object
     * An object you can define that will be sent along with the report.
     * Only string properties are sent correctly.
     * Nested objects are not supported.
 
-Developers are required to call this method before using other `crashReporter` APIs.
+Developers are required to call this method before using other `crashReporter`
```

Comment:
Maybe "You" here instead of "Developers"? "You" seems to be used elsewhere in this doc.